sw/device/sca/aes_serial/prng.c

```diff
@@ -106,7 +106,7 @@ static void init_by_array(uint32_t init_key[], int32_t key_length) {
 }
 
 /* generates a random number on [0,0xffffffff]-interval */
-static uint32_t genrand_int32(void) {
+__attribute__((noinline)) static uint32_t genrand_int32(void) {
```

Comment:
nit: Can you add a comment to describe why this is needed?

Reply:
Done.